test/Deployment.js

```diff
@@ -0,0 +1,101 @@
+const Contracts = require('./helpers/Contracts');
+const runDeployment = require('./helpers/runDeployment');
+
+const config =
+{
+    "reserves": [
```

Comment:
These shouldn't be quoted, since it's not a JSON file

Reply:
ah yeah, copy-paste (from the README file) mistake

Reply:
fixed